src/completers/kubectl.zsh

```diff
@@ -118,16 +118,16 @@ _fzf_complete_kubectl() {
         fi
     done
 
-    subcommands=($(_fzf_complete_parse_argument 2 1 "$arguments" "${(F)kubectl_options_argument_required}" || :))
+    subcommands=($(_fzf_complete_parse_argument 2 1 "${${(q)arguments[@]}}" "${(F)kubectl_options_argument_required}" || :))
```

Comment:
Nesting `${}` isn't useless. When there isn't nesting, the argument expands and doesn't match the arguments `_fzf_complete_parse_argument` requires.
```zsh
$ tmp() { local arguments=("${(Q)${(z)1}[@]}"); echo $#; echo $^arguments, }
$ tmp "${(q)v}"
1
kubectl  top  aaa,
$ tmp "${(q)v[@]}"
5
kubectl,
$ tmp "${${(q)v[@]}}"
1
kubectl, , top, , aaa,
```

Reply:
Yeah, this is obvious because once it expands without the extra `${}`, they occupy the number of elements in that array. It has to be wrapped with `${}` so it expands to a string. Totally makes sense!